sunpy/util/great_arc.py

```diff
@@ -0,0 +1,121 @@
+#
+# Calculates the co-ordinates along great arcs between two specified points
+# which are assumed to be on disk.
+#
+from __future__ import absolute_import, division, print_function
+import numpy as np
+import sunpy.coordinates
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+
+
+def great_arc(a, b, center=SkyCoord(0*u.km, 0*u.km, 0*u.km, frame='heliocentric'), num=100):
+    """
+    Calculate a user-specified number of points on a great arc between a start
+    and end point on a sphere.
+
+    Parameters
+    ----------
+    :param a: `~astropy.coordinates.SkyCoord`
```

Comment:
have you been reading too many codes using google doc style?! this should look like this:
```python
a : `~astropy.coordinates.SkyCoord`
    Start Point
b : `~astropy.coordinates.SkyCoord`
    end point
```